src/qml/NightstandPage.qml

```diff
@@ -0,0 +1,143 @@
+
+        key: "/desktop/asteroid/nightstand/brightness"
+        defaultValue: 30
+    }
+
+    ConfigurationValue {
+        id: nightstandDelay
+
+        key: "/desktop/asteroid/nightstand/delay"
+        defaultValue: 5
+    }
+
+    ConfigurationValue {
+        id: nightstandEnabled
+
+        key: "/desktop/asteroid/nightstand/enabled"
+        defaultValue: false
+    }
+
+    PageHeader {
+        id: title
+        text: qsTrId("id-nightstand-page")
+    }
+
+    Flickable {
+        anchors.fill: parent
+        contentHeight: onOffSettings.implicitHeight
+        boundsBehavior: Flickable.DragOverBounds
+        flickableDirection: Flickable.VerticalFlick
+        anchors.margins: Dims.l(15)
+
+        Column {
+            id: onOffSettings
+            anchors.fill: parent
+            LabeledSwitch {
+                //% "Nightstand enable"
+                text: qsTrId("id-nightstand-enable")
+                width: parent.width
+                height: Dims.l(20)
+                checked: nightstandEnabled.value
+                onCheckedChanged: nightstandEnabled.value = checked
+            }
+
+            Column {
+                width: parent.width
+                opacity: nightstandEnabled.value ? 1.0 : 0.4
+		enabled: nightstandEnabled.value
+                Label {
+                    //% "Nightstand Brightness"
+                    text: qsTrId("id-nightstand-brightness")
+                    font.pixelSize: Dims.l(6)
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                }
+                IntSelector {
+                    width: parent.width
+                    height: Dims.l(20)
+                    stepSize: 10
+                    value: nightstandBrightness.value
+                    onValueChanged: nightstandBrightness.value = value
+                }
+            }
+
+            Column {
+                width: parent.width
+                opacity: nightstandEnabled.value ? 1.0 : 0.4
+		enabled: nightstandEnabled.value
+                Label {
+                    //% "Nightstand delay"
+                    text: qsTrId("id-nightstand-delay")
+                    font.pixelSize: Dims.l(6)
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                }
+                IntSelector {
+                    width: parent.width
+                    height: Dims.l(20)
+                    stepSize: 5
+                    max: 30
+                    unitMarker: "s"
+                    value: nightstandDelay.value
+                    onValueChanged: nightstandDelay.value = value
+                }
+            }
+
+            Row {
+                width: parent.width
+                height: Dims.l(20)
+                opacity: nightstandEnabled.value ? 1.0 : 0.4
+		enabled: nightstandEnabled.value
+                Label {
+                    //% "select nightstand watchface"
```

Comment:
Uh for consistency could we have this start with a capital letter too ? :p 
```
    Nightstand
Enable
Brightness
Delay
Select watchface ->
```

Reply:
Done